### PR TITLE
conditional checking if campaigns doesnt have drupal_id

### LINKED ISSUE #81: User show fails if no campaigns
I'm getting an error on the first "test" user in prod

`cURL error 6: Could not resolve host: campaigns`
thinking maybe it's failing because the user doesn't have any campaigns, or the campaigns they are signed up for don't exist on drupal

### PR DESCRIPTION
**Pull request by axton21**
Fixes #81 @angaither @DFurnes @jonuy 

Talking about old northstar data that doesnt gives back drupal_id with @jonuy.
Hopefully this will solve the problem for now.

### CODE REVIEW

File: app/controllers/UsersController.php [hunk 1]
```diff
@@ -67,9 +67,11 @@ public function show($id)
       $aurora_user = User::where('_id', $id)->first();
       if (!empty($user['campaigns'])){
         foreach($user['campaigns'] as $campaign){
-          array_push($campaigns, $this->drupal->getCampaign($campaign['drupal_id']));
-          if (!empty($campaign['reportback_id'])) {
-            array_push($reportbacks, $this->drupal->getReportbacks($campaign['reportback_id']));
+          if (!empty($campaign['drupal_id'])) {
+            array_push($campaigns, $this->drupal->getCampaign($campaign['drupal_id']));
```
Comment (angaither) [inline app/controllers/UsersController.php]: what if `getCampaign` fails? I think that may be the case for some of the test data

Comment (axton21) [inline app/controllers/UsersController.php]: Yup, working on it =)
Need to make another filter in `drupal->getCampaign`

Comment (angaither) [inline app/controllers/UsersController.php]: Okay, that can be a different PR!

**Comment by angaither** [comment]
:shipit: :rocket:

**Pull request merged**